pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -212,12 +211,14 @@ func (s *DiscoveryServer) updateClusterInc(push *model.PushContext, clusterName
 	// arbitrary destination rule's subset labels!
 	labels := push.SubsetToLabels(nil, subsetName, hostname)
 
+	var dummySidecarScope *model.SidecarScope
```

Comment:
Seems fishy... Why is this a null pointer and passed around ?

Reply:
I agree, I'll change this